File: app/src/main/java/com/bigstone/expresswaynav/base/BaseActivity.java
package com.bigstone.expresswaynav.base;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.fragment.app.FragmentActivity;

import com.bigstone.expresswaynav.utils.StatusBarHelper;

public class BaseActivity extends FragmentActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        StatusBarHelper.translucent(this);
        StatusBarHelper.setStatusBarLightMode(this);
    }
}

File: app/src/main/java/com/bigstone/expresswaynav/base/BaseActivity.kt
package com.bigstone.expresswaynav.base

import android.os.Bundle
import androidx.fragment.app.FragmentActivity
import com.bigstone.expresswaynav.utils.StatusBarHelper

abstract class BaseActivity : FragmentActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        StatusBarHelper.translucent(this)
        StatusBarHelper.setStatusBarLightMode(this)
    }
}